### PR TITLE
Enable logging when running app with gunicorn

### PR DESCRIPTION
**Pull request by alexander871015**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
         logging.basicConfig(level=default_level)
     logging.info('Successfully initialized logger.')
 
+#move everything down so init_loggin is defined
```
Comment (azhu7) [inline main.py]: Please follow the comment format:
`# Start with capital letter, end with period.`

Comment (azhu7) [inline main.py]: You probably don't need this comment.

File: main.py [hunk 2]
```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
...
+init_logging()#start logging bfore creating the application so that __init__.py log will stay in logging_dev/loggin_prod
```
Comment (azhu7) [inline main.py]: Move the `# Start logging` comment to the line above `init_logging()`
Also there is a typo in the comment. And maybe word as `... __init__.py will log to logging_dev/logging_prod.`

**Review by azhu7 — COMMENTED** [review]
Please change the PR description to a title that someone else without context could understand. For example: 'Enable logging when running app with gunicorn.`
Then you can go into more detail about the fix in your PR description.
As for testing it, probably run the app with and without your fix and see that the logs show only after you apply the fix.

File: main.py (hunk 2 above)
Comment (alexander871015) [reply main.py]: What do you mean by word as?

File: main.py [hunk 3]
```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
         logging.basicConfig(level=default_level)
     logging.info('Successfully initialized logger.')
 
+#Start logging bfore creating the application so that __init__.py log will stay in logging_dev/loggin_prod.
```
Comment (azhu7) [inline main.py]: Put a space between # and first word (a rule of thumb is to follow the formatting of the rest of the project. Also typo bfore. Also please reword as `__init__.py will log to logging_dev/logging_prod.`

**Review by azhu7 — DISMISSED** [review]

**Pull request merged**